packages/trader/src/Modules/Trading/Components/Form/TradeParams/trade-type-tabs.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import React from 'react';
+import { ButtonToggle } from '@deriv/components';
+import { isTurbosContract, isVanillaContract } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+
+type TTradeTypeTabs = {
+    className?: string;
+};
+
+const TradeTypeTabs = observer(({ className }: TTradeTypeTabs) => {
+    const {
+        modules: { trade },
```

Comment:
@maryia-deriv `trade` is `any` so we are not getting any TS check with the `useStore` hook, You would need to use the changes from @hamza-deriv [PR](https://github.com/binary-com/deriv-app/pull/7922/files/0577d9750932e7626205b69a0ba787542a9e4a91#diff-b086cf4b359e17670331fc583edb630316c89a53304cd087641853e04a14dc6f) to add types for `trade`.

After making the changes, Since the store itself is in JS, Some types might be inferred as `any`, For those please override the type in the declaration file 🙏🏻

Reply:
@farzin-deriv by right useStore has to return all types we need for the stores that have to have types, we should not add types for stores in every files they are used. Let's wait until stores are typed and useStore returntype will be successfully inferred in this file as well as other files where useStore and stores are used. Adding types to stores and useStore is out of the PR' scope.
cc: @matin-deriv @balakrishna-deriv